rust/ql/test/query-tests/diagnostics/SummaryStats.expected

```diff
@@ -1,5 +1,9 @@
 | Elements extracted | 210 |
 | Elements unextracted | 0 |
-| Files extracted | 6 |
+| Extraction errors | 6 |
+| Extraction warnings | 0 |
```

Comment:
I would expect `6` warnings and `0` errors, why is it the other way around in the summary?

Reply:
Ah, it looks like I hadn't updated the `.expected` file.  Now fixed.